### PR TITLE
Perfect documentations

### PR DESCRIPTION
**Pull request by yuxiaobo96**
Signed-off-by: yuxiaobo <yuxiaobogo@163.com>

### Ⅰ. Describe what this PR did
Perfect documentations.

### Ⅱ. Does this pull request fix one issue?

NONE

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)

Don't need.

### CODE REVIEW

File: pkg/errortypes/supernode_errors.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ var (
 
 	// ErrUnknowError represents the error should not happen
 	// and the cause of that is unknown.
-	ErrUnknowError = DfError{codeUnknowError, "unknow error"}
+	ErrUnknowError = DfError{codeUnknowError, "unknown error"}
```
Comment (yeya24) [inline pkg/errortypes/supernode_errors.go]: Should it be ErrUnknownError?

**Comment by yuxiaobo96** [comment]
/cc @Starnop 
Thank you.

File: pkg/errortypes/supernode_errors.go (hunk 1 above)
Comment (yeya24) [reply pkg/errortypes/supernode_errors.go]: @yuxiaobo96  If you want to address the typo, please address all of them such as `ErrUnknowError`, `codeUnknowError`

Comment (yuxiaobo96) [reply pkg/errortypes/supernode_errors.go]: Ok,

**Comment by yuxiaobo96** [comment]
I have added them.

File: dfget/core/regist/register.go [hunk 2]
```diff
@@ -105,7 +105,7 @@ func (s *supernodeRegister) checkResponse(resp *types.RegisterResponse, e error)
 		return errortypes.New(constants.HTTPError, e.Error())
 	}
 	if resp == nil {
-		return errortypes.New(constants.HTTPError, "empty response, unknown error")
+		return errortypes.New(constants.HTTPError, "empty response, ErrUnknowError")
```
Comment (yeya24) [inline dfget/core/regist/register.go]: Why did you change the error message?

Comment (yuxiaobo96) [reply dfget/core/regist/register.go]: Because of your problem with the way you express.

Comment (yuxiaobo96) [reply dfget/core/regist/register.go]: I can't understand what you really mean.

Comment (yuxiaobo96) [reply dfget/core/regist/register.go]: Or you are not clear about what you said.

Comment (Starnop) [reply dfget/core/regist/register.go]: 😢

Comment (yuxiaobo96) [reply dfget/core/regist/register.go]: I am sorry.
